optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/stream/ConstraintStreamScoreDirector.java

```diff
@@ -172,4 +172,11 @@ public void afterProblemFactRemoved(Object problemFact) {
         super.afterProblemFactRemoved(problemFact);
     }
 
+    // ************************************************************************
+    // Getters/setters
+    // ************************************************************************
+
+    public ConstraintSession<Solution_, Score_> getSession() {
+        return session;
+    }
```

Comment:
Don't like it, but I needed this for the test.
This is private API, so we should be good.

Reply:
Ok. Someday we need to talk about merging ConstraintStreamSession and ConstraintStreamScoreDirector to avoid unneeded indirection :)